in_toto/model_test.go

```diff
@@ -719,7 +719,8 @@ func TestValidatePubKey(t *testing.T) {
 		Scheme: "rsassa-pss-sha256",
 	}
 
-	if err := validateRSAPubKey(testKey); err != nil {
+	err := validateKey(testKey)
+	if !errors.Is(err, ErrEmptyKeyField) {
```

Comment:
This one fails because the missing `KeyIdHashAlgorithms`, right?

Reply:
Yes, this is correct. Do we want this behavior? Or is the `keyIdHashAlgorithms` field optional? I remember, that we had another discussion around it, that all information is in the `scheme` field already..

Reply:
In in-toto/sslib it's optional. I'm fine either way (for the time being), with a preference towards making it optional. In the long run we want to get rid of the field anyway.

The corresponding discussion can be found i.a. in https://github.com/theupdateframework/tuf/issues/848, but it's not related to the scheme field. 

(The discussion who might think of is about whether it needs both the `keytype` and `scheme` fields, i.a. see https://github.com/secure-systems-lab/securesystemslib/issues/251)

Reply:
I made the field now optional in: https://github.com/in-toto/in-toto-golang/pull/56/commits/fa94594c5d8a77fbd96d5839efe8d121065e81f8